webui/react/src/components/LogViewer/LogViewer.tsx

```diff
@@ -564,6 +572,26 @@ const LogViewer: React.FC<Props> = ({
     [dateTimeWidth],
   );
 
+  const handleKeyboardScrolling = useCallback((e: KeyboardEvent) => {
```

Comment:
I think scrolling works without these code. just `tabIndex` is good.
Do you have reasons to keep it?

Reply:
Yes, I know it's for disabling the focus border. If we do that though, someone could press an arrow key a bunch, see that nothing is visibly happening, and assume we don't support scrolling via arrow keys.

Without the handleKeyboardScrolling I'm seeing that Page Up/Down and Home/End work, but arrow keys don't.

Reply:
🤔 i tested it with arrow key (up/down), and it works fine.

Reply:
https://user-images.githubusercontent.com/109113616/196539750-cc24c5b1-b701-4810-8dab-e98a79f6b1d1.mov

Reply:
Looks like it works in Chrome but not in Firefox. I'll look into the differences in how they handle scrolling.

Reply:
Actually apparently I turned on a weird setting in Firefox. Removed this.

Reply:
I completely forgot about Firefox. It still works well on Firefox tho, but it's kinda not working on Safari.

Reply:
Without the `handleKeyboardScrolling` I don't think removing the outline is necessary though, since we're not manually setting focus.